spec/cypress/e2e/media_spec.cy.archive.js

```diff
@@ -112,7 +112,7 @@ describe("Media", () => {
         cy.contains("Ich bestätige hiermit, dass durch die Veröffentlichung des Mediums auf der MaMpf-Plattform keine Rechte Dritter verletzt werden.").click();
         cy.get("#publishMediumModal").contains("Speichern").click();
         cy.wait(100);
-        cy.visit(`lectures/${lectures[0].id}/food?project=kaviar`);
+        cy.visit(`lectures/${lectures[0].id}/food?project=lesson_material`);
```

Comment:
Shouldn't this `?project=` route not exist anymore? Same for the other changes in this file.

Reply:
Indeed. Fixed.